templates/Procfile

```diff
@@ -1,2 +1,2 @@
-web: bundle exec unicorn -p $PORT -c ./config/unicorn.rb
+web: bundle exec rails server Puma -p ${PORT:-3000} -c ./config/puma.rb
```

Comment:
> the example Procfile in the Heroku docs uses puma directly. Of course, the downside to that solution is losing logging...

@pedrosmmoreira Curious what you mean by "losing logging"? The Rails app itself will still log to whatever mechanism you've set the log to (`STDOUT` when running on Heroku). Are there other logs I'm not aware of or missing?